packages/x-date-pickers/src/StaticDatePicker/StaticDatePickerKeyboard.test.tsx

```diff
@@ -46,6 +46,8 @@ describe('<StaticDatePicker /> keyboard interactions', () => {
         // eslint-disable-next-line material-ui/disallow-active-element-as-key-event-target
         fireEvent.keyDown(document.activeElement!, { keyCode, key });
 
+        // static analysis can't calculate the final grid cell label connected with day of week `columnheader`
+        // resulting in `<Week day> <number>` being the actual accessible name for each picker day
```

Comment:
I don't understand what does this comment means

Reply:
I tried communicating the fact that `getAccessibleName` can't correctly return the actual label that is spoken due to various roles.  
Screen reader actually always says the day you are focusing as well as the week day, referenced by the element with `columnheader` role.

Reply:
You mean in the test we verify that the accessible name is "4" but screen-reader will say "Monday 4" which is the correct behavior ?

Reply:
Yup, that's exactly the issue I was trying to explain with this comment. 🙈 
Clearly, it didn't work well enough. 😆 
Do you have any suggestions for a better comment copy?

Reply:
What about

> Based on column header, screen reader should prononce <Week day> <day number>
But `getAccessibleName` does not do the link between column header and cell value, so we only get <day number> in test